test/parallel/test-stream-buffer-list.js

```diff
@@ -34,7 +34,8 @@ assert.deepStrictEqual(list, new BufferList());
 
 const tmp = util.inspect.defaultOptions.colors;
 util.inspect.defaultOptions = { colors: true };
+const color = util.inspect.colors[util.inspect.styles.number];
 assert.strictEqual(
   util.inspect(list),
-  'BufferList { length: \u001b[33m0\u001b[39m }');
```

Comment:
does anyone know why this test needs to be done with colors?

Reply:
There is a custom inspect function on the `BufferList`. It is special in the way that it should set the color on that property in case `colors = true`.

Reply:
I meant why is it testing that colors work, that's effectively testing util, not bufferlist

Reply:
No, it is testing the return value of the custom inspect function of `BufferList`.